framework/auth/core.py

```diff
@@ -93,6 +94,15 @@ def validate_personal_site(value):
 def validate_social(value):
     validate_personal_site(value.get('personal'))
 
+
+def validate_email(item):
```

Comment:
I'm OK with it as-is if you are. It's as minimal as possible, really only checking to see if it "looks like" an email. I figure the only way to verify that it *is* an email is to confirm it, which we're doing :)